### ISSUE TITLE
Quantized tanh does not correspond to the tanh used in tensorflow

### ISSUE THREAD

**Issue by laurilaatu** [google/qkeras#62]
The quantized tanh function defined using sigmoid does not correspond very well to the tanh function used in tf.keras.activations.tanh. A possibility to have better corresponding tanh function would be useful in some use-cases.

`set_internal_sigmoid("real")`
`plt.plot(np.arange(-5,5,0.01), qkeras.quantized_tanh(32)(np.arange(-5,5,0.01)).numpy())`
`plt.plot(np.arange(-5,5,0.01), tf.keras.activations.tanh(np.arange(-5,5,0.01)).numpy())`

![tf_qkeras_activations](https://user-images.githubusercontent.com/19837180/109654292-52bc0f80-7b62-11eb-8f60-d48ba917e5b1.png)
